bin/lib/library_yaml.py

```diff
@@ -0,0 +1,23 @@
+import os
+import yaml
+from pathlib import Path
+
+from lib.config_safe_loader import ConfigSafeLoader
+
+class LibraryYaml:
+    def __init__(self, yaml_dir):
+        self.yaml_dir = yaml_dir
+        self.yaml_path = Path(os.path.join(self.yaml_dir, 'libraries.yaml'))
+        self.Load()
+
+    def Load(self):
```

Comment:
can we use python-style naming convention please (horrid `snake_case` but at least consistent with all the rest of the code! thanks!)

Reply:
oops, yes